src/main/java/WayofTime/alchemicalWizardry/client/nei/NEIMeteorRecipeHandler.java

```diff
@@ -40,7 +42,9 @@ public CachedMeteorRecipe(MeteorParadigm meteor, ItemStack focusStack) {
             int row = 0;
             int col = 0;
 
-            float totalMeteorWeight = meteor.getTotalMeteorWeight();
+            float totalComponentWeight = meteor.getTotalComponentWeight();
+            float totalFillerWeight = meteor.getTotalFillerWeight();
```

Comment:
Very small nitpick, either move `totalFillerWeight` down to where you get `sortedFiller` (i.e., initialize all variables where they are needed), or move `sortedFiller` here (i.e., initialize everything at the start).

Reply:
Addressed in [49e0274](https://github.com/GTNewHorizons/BloodMagic/pull/60/commits/49e0274d9ea18e2c9db6679ef36285538417cc92)